src/js/tracks/text-track-cue-list.js

```diff
@@ -30,12 +30,16 @@ let TextTrackCueList = function(cues) {
     }
   });
 
-  if (VjsLib.IS_IE8) {
+  if (Lib.IS_IE8) {
     return list;
   }
 };
 
 TextTrackCueList.prototype.setCues_ = function(cues) {
+  let oldLength = this.length || 0;
```

Comment:
This function got screwed up with the let changes so I had to revert it to get tests to pass.